packages/devtools_app/lib/src/framework/scaffold.dart

```diff
@@ -315,17 +315,20 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
             serviceConnection.serviceManager.connectedAppInitialized &&
                 !offlineDataController.showingOfflineData.value &&
                 _currentScreen.showConsole(widget.embedMode);
-
+        final containsSingleSimpleScreen =
```

Comment:
OOC, what is a `SimpleScreen`? Is that just a default DevTools as opposed to an Extension screen?

Reply:
SimpleScreen is what we use for displaying rudimentary screens that are basically just a single child body: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/framework/scaffold.dart/#L425. The extends the Screen class though, so it lets us show simple content as part of the DevTools scaffolding structure. This is used by the `DevToolsScaffold.withChild` constructor for things like page not found messages, displaying a single offline screen, etc.